lib/internal/bootstrap/loaders.js

```diff
@@ -90,9 +90,9 @@
 
   // Minimal sandbox helper
   const ContextifyScript = process.binding('contextify').ContextifyScript;
-  function runInThisContext(code, options) {
-    const script = new ContextifyScript(code, options);
-    return script.runInThisContext();
+  function runInThisContext(code, filename) {
+    const script = new ContextifyScript(code, filename);
```

Comment:
At this point do we even need this helper? Maybe just inline it in the try-catch block?